workflow/automation/estimation/estimate_cybershake.py

```diff
@@ -415,36 +415,33 @@ def display_results(df: pd.DataFrame, verbose: bool = False):
 
     print()
     sum_df = df.sum()
-    header = "{:<12}{:<10}{:<8}".format("core hours", "run time", "")
+    header = "{:>20}{:>10}".format("core hours", "run time")
     print(process_types)
```

Comment:
This print is duplicated within the if / outside.

I think I'd prefer process_types to be called `header` and header to be called `sub_header`.

Reply:
It's just tricky as one of them has cores and the other doesn't so I can't just specify the subheader outside of the loop for both. I can rename them though

Reply:
Ah, I was just talking about the print(process_types) one. The other "sub-header" is all good being conditional.

Reply:
Ok, I'm not sure how it can be outside of the loop though? Since when verbose is set it needs to print twice and when it's not set it only prints once. Unless I'm thinking about this wrong?

Reply:
Oh yep. I'm the crazy one. As is - you're all good.